R.NET/Function.cs

```diff
@@ -125,5 +125,30 @@ private SymbolicExpression createCallAndEvaluate(IntPtr argument)
               }
           }
       }
+
+      /// <summary>
+      /// Invoke the function with optionally named arguments by order.
+      /// </summary>
+      /// <param name="args">one or more tuples, conceptually a pairlist of arguments.
+      /// The argument names must be unique; null or empty string indicates unnamed argument. </param>
+      /// <returns>The result of the function evaluation</returns>
+      public SymbolicExpression InvokeNamedFast(params Tuple<string, SymbolicExpression>[] args)
+      {
+          IntPtr argument = Engine.NilValue.DangerousGetHandle();
+          var rfInstall = GetFunction<Rf_install>();
+          var rSetTag = GetFunction<R_SetExternalPtrTag>();

```

Comment:
The function call is somewhat misleading, there are actually no external pointers involved. We are setting tags to the argument list which is never passed outside.
The R source code example (tests/Embedding/RNamedCall.c) actually calls `SET_TAG` for setting argument names. Unfortunately this function is not exported and the most relevant I can get is `R_SetExternalPtrTag`. However after digging into those two functions I found they are actually the same and I think it's safe to just use the external one.
Here's the body of `SET_TAG` and `R_SetExternalPtrTag` extracted from memory.c:
```c
void (SET_TAG)(SEXP x, SEXP v) { FIX_REFCNT(x, TAG(x), v); CHECK_OLD_TO_NEW(x, v); TAG(x) = v; }

#define EXTPTR_TAG(x)	TAG(x)
void R_SetExternalPtrTag(SEXP s, SEXP tag)
{
    FIX_REFCNT(s, EXTPTR_TAG(s), tag);
    CHECK_OLD_TO_NEW(s, tag);
    EXTPTR_TAG(s) = tag;
}
```